### ISSUE TITLE
ResNet-50 lr_policy: “step” doesn't work, always divide by 10

### ISSUE THREAD

**Issue by ivy94419** [BVLC/caffe#5706]
I'm using ResNet-50 of Caffe in order to classify images. I can finetune the it using ResNet-50_model.caffemodel, and the inital learning rate I set is 0.001, but the learning rate after each 500 iter is divide by 10 instead of reduce by 0.00005.
Here is my resnet_solver.prototxt

```
net: "data/resnet_data/resnet_50.prototxt"

test_iter: 320
test_interval: 100

base_lr: 0.001
lr_policy: "step"
gamma: 0.1
stepsize: 500
display: 100
max_iter: 2000
momentum: 0.9
weight_decay: 0.0005
snapshot: 500

snapshot_prefix: "data/resnet_data/model/resnet_grid"
solver_mode: GPU
```
And result shows:

 iteration 100, lr = 0.001
 iter 500, lr = 0.0001
 iter 1000 lr = 1e-05
 and so on...

How can I solve it?

**Comment by madongyu** [comment]
you use learning policy of "step"，It read the variable of **stepsize** rather than **weight_decay**

**Comment by ivy94419** [comment]
@madongyu I got that, thanks!
